lib/gems/pending/metadata/MIQExtract/MIQExtract.rb

```diff
@@ -158,10 +158,10 @@ def extract(category, &blk)
   end
 
   def getVMConfig(_c)
-    # Get VM config in XML format
-    config_xml = @target.vmConfig.toXML(true, @target)
-
     begin
+      # Get VM config in XML format
+      config_xml = @target.vmConfig.toXML(true, @target)
```

Comment:
You can't just raise or log an error here. Errors in the original block SHOULD be silent, it's only the new call added to the block that shouldn't be silent.

Here again, this should all be moot because the call to vmConfig should work.